themes/default/content/blog/pulumi-backstage-plugin/index.md

```diff
@@ -0,0 +1,99 @@
+
+## Pulumi Joins the Stage
+
+Today we are announcing a new [Pulumi Plugin](https://github.com/pulumi/pulumi-backstage-plugin) for [Backstage](https://backstage.io/). Using the Pulumi Backstage Plugin, you can create a new Pulumi project from an organization template stored in Pulumi, you can run previews and updates, and you can run and view Deployment.
+
+![Pulumi Backstage Plugin Activity](./pulumi_backstage_plugin_activity.png)
+
+## How to get started with the Pulumi Backstage Integration
+
+![Pulumi Backstage Plugin Icon](./pulumi_backstage_plugin_icon.png)
+
+The Pulumi Backstage integration has two independent components:
+
+* Pulumi Scaffolder Backend Module
+* Pulumi Backstage Plugin
+
+## What is Scaffolding in Backstage?
+
+The Backstage Scaffolder is a core part of the Backstage developer portal. Using standardized templates, developers can create new components (services, libraries, websites, or other software).  With a few clicks, developers can create new software projects provisioned with best practices, with code managed in the right repositories, and with the right configurations, all from a self-service UI.
+
+## Pulumi Extends Backstage Scaffolding
+
+The **Pulumi Scaffolder Backend Module** extends the scaffolding with two new actions: **pulumi:new and pulumi:up**
+
+With the new **Pulumi Scaffolder Backend Module**, organizations can standardize infrastructure creation through their own custom templates. This enables a consistent approach for every new project while minimizing manual setup and configuration by developers and platform teams.
+
+The next component of the Pulumi Backstage integration is the **Pulumi Plugin**. Backstage Plugins are extensions or add-ons to the core Backstage application that allow you to integrate additional tools, services, and functionalities into the platform, enhancing its capabilities and customizing it to meet the specific needs of your development environment.
+
+The Pulumi plugin extends Backstage by displaying essential details like the Pulumi stack, organization, project name, and description, as well as providing an activity view with aggregated information from the Pulumi Cloud.
+
+## Connecting Pulumi Cloud with Backstage with the Pulumi Backstage plugin
+
+The Pulumi plugin extends Backstage by displaying essential details like the Pulumi stack, organization, project name, and description, as well as providing an activity view for the entity with aggregating information from the Pulumi Cloud.
+
+![Pulumi Backstage Plugin System](./pulumi_backstage_plugin_system.png)
+![Pulumi Backstage Plugin Component](./pulumi_backstage_plugin_component.png)
+
+Utilizing the **Pulumi Backstage** integration, including its plugin and scaffolder, brings forth many benefits that streamline infrastructure management, enhance developer experience, and foster a well-orchestrated DevOps culture. See the key benefits below.
+
+## Key Benefits of the Pulumi Backstage Plugin
```

Comment:
@cnunciato Should the items in here be bulleted instead of or in addition to bold emphasis?